e2e-polybft/cardanofw/test-apex-user.go

```diff
@@ -256,7 +260,7 @@ func BridgeAmountFullMultipleReceiversNexus(
 	t *testing.T, ctx context.Context, txProvider wallet.ITxProvider, networkConfig TestCardanoNetworkConfig,
 	multisigAddr, feeAddr string, sender wallet.IWallet,
 	receiverAddrs []string, sendAmount uint64,
-) string {
+) (string, error) {
 	t.Helper()
```

Comment:
Why is this function t.Helper and why it receives `*testing.T` if all errors are returned as it is normal function?

Reply:
my bad,  I had to change it so that it returns error, since we need it for some tests. Forgot to remove this part. I will update it.